docs/admin/rescheduler.md

```diff
@@ -0,0 +1,58 @@
+---
+assignees:
+- davidopp
+- filipg
+- piosz
+
+---
+
+* TOC
+{:toc}
+
+# "Guaranteed" scheduling of critical addon pods
```

Comment:
Addon -> add-on

Probably should be a global replacement throughout the document.